pre-docs/writing_plans.md

```diff
@@ -60,8 +55,10 @@ Specify each parameter in your plan with its data type. For example, you might
 want parameters to specify which nodes to run different parts of your plan on.
 
 The following example shows node parameters specified as data type
-`Variant[String[1], Array[String[1]]]`. For more information about these data
-types, see the common data types table in the related metadata type topic.
+`TargetSpec` this allows this parameter to be passed as a single url, comma
+seperated url lis, Target datatype or Array of either. For more information
```

Comment:
`lis` -> `list`

Reply:
Also need a comma after `datatype`.

Reply:
Are we using `datatype` or `data type`? Doesn't seem consistent.

Reply:
we use data type on https://puppet.com/docs/puppet/5.3/lang_data_type.html